src/@ionic-native/plugins/notificare/index.ts

```diff
@@ -0,0 +1,723 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+
+/**
+ * @name Notificare
+ * @description
+ * Official Ionic Native plugin for Notificare.
+ *
+ * @usage
+ * ```typescript
+ * import { Notificare } from '@ionic-native/notificare';
```

Comment:
Change the import to `import { Notificare } from '@ionic-native/notificare/ngx';`